apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| decimals                          | bigint | Number of decimal places for token value, typically for fungible tokens. Ex. 18                                                   |
+| icon_uri                          | String | URI for the icon of the asset. Ex. "https://cdn.example.com/icons/123"                                                            |
+| last_transaction_timestamp        | String | Timestamp of the last transaction involving this asset. Ex. "2024-04-17T02:14:25.68771"                                           |
+| last_transaction_version          | bigint | Blockchain version of the last transaction involving this asset. Ex. 10000000                                                     |
+| name                              | String | The formal name of the asset. Ex. "Digital Gold"                                                                                  |
+| project_uri                       | String | URI linking to the project information associated with this asset. Ex. "https://www.example.com/project_name/"                    |
+| supply_aggregator_table_handle_v1 | String | Legacy handle for the supply aggregator table from an earlier schema version. Ex. "handle_67890"                                  |
+| supply_aggregator_table_key_v1    | String | Legacy key for accessing the supply aggregator table in earlier schema versions. Ex. "key_12345"                                  |
+| symbol                            | String | The trading symbol of the asset. Ex. "DGOLD"                                                                                      |
+| token_standard                    | String | Standard that the asset adheres to. Ex. "v1"                                                                                      |
+
+### `fungible_asset_activities`
+
+This tracks the activity of fungible assets. It includes v1 token data.
+
+| Field                       | Type    | Description                                                                                                                                          |
+| --------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| amount                      | bigint  | The amount of the asset involved in the activity. Ex. 1000                                                                                           |
+| asset_type                  | String  | The type of the asset, described by a Move resource. Ex. "0x1::aptos_coin::AptosCoin"                                                                |
+| block_height                | bigint  | The blockchain height at which this activity occurred. Ex. 1500000                                                                                   |
```

Comment:
The block ID.

Reply:
Do you mean that this description should be include "block id" instead of "blockchain height"? (Or what do you mean by The block ID?)